resources/eds_templates/diff_report/config.json

```diff
@@ -2,7 +2,7 @@
     "template": "main.html",
     "conf":
     {
-        "default_endpoint" : "http://localhost:3030/cob/sparql",
+        "default_endpoint" : "http://localhost:3030/report1/sparql",
```

Comment:
this is a redundant change as this serves only as an example, the data from there not being used.